### ISSUE TITLE
InvalidArgumentException using the new release GRAV 1.0.0 rc4

### ISSUE THREAD

**Issue by dvago** [giansi/grav-plugin-cookies-policy#6]
Just enabled the plugin from the latest version of grav admin 1.0.0 rc4

When I save the configuration (using the admin panel) I get the following error:

InvalidArgumentException
HELP
The Cookie Policy type variable value must be one of "bar" or "dialog". You gave "normal"

Cookiepolicy.php from line 42

``` php
 throw new \InvalidArgumentException('The Cookie Policy type variable value must be defined. At the moment it is empty. If you are overriding the default confoguration, please define the type variable too with a valid value.');
        }

        if (!preg_grep("/" . $type . "/i", array(
            "bar",
            "dialog",
        ))){
            throw new \InvalidArgumentException(sprintf('The Cookie Policy type variable value must be one of "bar" or "dialog". You gave "%s"', $type));
        }
```

Could you please have a look?

**Comment by giansi** [comment]
Hi,
just tried again with the latest Grav but everything works fine here. I've changed all options and I did not get any error. Could you check your configuration?

**Comment by giansi** [comment]
Ok, got it.

**Comment by giansi** [comment]
Fixed with 1.0.6 release. Thank you
